src/Widgets/NotificationEntry.vala

```diff
@@ -48,40 +57,54 @@ public class Notifications.NotificationEntry : Gtk.ListBoxRow {
 
         var app_image = new Gtk.Image () {
             icon_name = app_icon,
-            pixel_size = 48
+            pixel_size = 48,
+            margin = 6,
+            margin_right = 0
+        };
+
+        var title_label = new Gtk.Label ("<b>%s</b>".printf (fix_markup (notification.summary))) {
+            ellipsize = Pango.EllipsizeMode.END,
+            hexpand = true,
+            width_chars = 27,
+            max_width_chars = 27,
+            use_markup = true,
+            xalign = 0
         };
 
-        var title_label = new Gtk.Label ("<b>%s</b>".printf (fix_markup (notification.summary)));
-        title_label.ellipsize = Pango.EllipsizeMode.END;
-        title_label.hexpand = true;
-        title_label.width_chars = 27;
-        title_label.max_width_chars = 27;
-        title_label.use_markup = true;
-        title_label.xalign = 0;
 
-        var time_label = new Gtk.Label (Granite.DateTime.get_relative_datetime (notification.timestamp));
+        var time_label = new Gtk.Label (Granite.DateTime.get_relative_datetime (notification.timestamp)) {
+            margin_right = 6
+        };
         time_label.get_style_context ().add_class (Gtk.STYLE_CLASS_DIM_LABEL);
 
         var grid = new Gtk.Grid () {
+            hexpand = true,
             column_spacing = 6,
-            margin = 6
+            margin = 12,
+            margin_top = 1,
+            margin_bottom = 11
```

Comment:
Looks like you're trying to fix some off-by-one here. Can you leave a comment about what that is?

Reply:
I made a quick comparison:
left: margin_bottom = 11, margin_top = 1
mid: margin_bottom = 12, margin_top = 0
right: margin_bottom = 12, margin_top = 12
![Untitled](https://user-images.githubusercontent.com/43096739/94495190-6816c180-01f1-11eb-854f-306f3d690782.png)
In my opinion the second option is the best, because the space over the first notification isn't as big and the notification isn't cropped. 
I'm not sure if there is a better way to fix the cropping.